winui/Rating/Overview.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About WinUI Rating control| Syncfusion
+title: About WinUI Rating control | Syncfusion
 description: Learn here all about introduction of Syncfusion WinUI Rating Control (SfRating) with key features and more.
```

Comment:
description: Learn all about the introduction of Syncfusion WinUI Rating Control (SfRating) with essential features and more here.

Reply:
Changed the description content as mentioned above.